### PR TITLE
n19761: add notes and questions

### PR DESCRIPTION
**Pull request by jnewbery**
Thanks to @fanquake for the questions. I've added some very brief notes.

### CODE REVIEW

File: _posts/2020-09-23-#19761.md [hunk 1]
```diff
@@ -10,14 +10,44 @@ status: upcoming
...
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core requires several
+  [dependencies](https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md).
+  These include large dependencies such as [boost](https://www.boost.org/),
+  [libevent](https://libevent.org/) and [qt](https://libevent.org/), as well as
```
Comment (jonatack) [inline _posts/2020-09-23-#19761.md:18]: qt link -> https://www.qt.io

File: _posts/2020-09-23-#19761.md [hunk 2]
```diff
@@ -10,14 +10,44 @@ status: upcoming
...
+  several smaller dependencies.
+
+- These dependencies can be built locally as part of a [depends
+  build](https://github.com/bitcoin/bitcoin/blob/master/depends/README.md).
+
+- The versions of the dependency libraries used is pinned by a specific hash
```
Comment (jonatack) [inline _posts/2020-09-23-#19761.md:24]: s/is/are/ or s/versions/version/ or "The version of each dependency library"

File: _posts/2020-09-23-#19761.md [hunk 3]
```diff
@@ -10,14 +10,44 @@ status: upcoming
...
+  (eg
```
Comment (jonatack) [inline _posts/2020-09-23-#19761.md:25]: e.g.

File: _posts/2020-09-23-#19761.md [hunk 4]
```diff
@@ -10,14 +10,44 @@ status: upcoming
...
+  [here](https://github.com/bitcoin/bitcoin/blob/652c45fdbbd55bde95c8c6cf08a5feb6055ac112/depends/packages/bdb.mk#L5)
+  for bdb).
+
+- Sometimes small changes need to be made to the library before building. This
+  has previously been done using [sed](https://en.wikipedia.org/wiki/Sed). sed
```
Comment (jonatack) [inline _posts/2020-09-23-#19761.md:30]: - suggest s/sed/`sed`/ for the second "sed" that starts the third sentence (as well as for `sed` in the following bullet point); `sed` is already in `code` format in the questions

- and/or start a new bullet point for the third sentence, for readability

**Review by jonatack — COMMENTED** [review]
Thanks @fanquake and @jnewbery! I think this is very useful info and that people will learn a lot (I definitely will).

**Comment by jnewbery** [comment]
Thanks for the review @jonatack !

**Pull request merged**